components/Dialogs/examples/DialogsAlertViewController.m

```diff
@@ -47,11 +46,7 @@ - (void)collectionView:(UICollectionView *)collectionView didSelectItemAtIndexPa
   }
 }
 
-
-
 - (IBAction)didTapShowAlert:(id)sender {
-  [[MDCButton appearanceWhenContainedIn:[MDCAlertController class], nil]
```

Comment:
Typically this is done in appDidFinishLoading.
In this case, I would do it in viewDidLoad.